src/core_plugins/console/public/src/sense_editor/mode/worker/worker.js

```diff
@@ -1817,6 +1817,7 @@ ace.define('ace/lib/es5-shim', ['require', 'exports', 'module'], function () {
     return Object(o);
   };
 });
+//do we need translate this syntax Error and how?
```

Comment:
@bmcconaghy do we need translate this syntax Error ?

Reply:
Yeah I think this needs to be translated.